TestCases/compliance-level-3/0072-feel-in/0072-feel-in.dmn

```diff
@@ -0,0 +1,2178 @@
+    </decision>
+-->
+
+    <decision name="dateTime_014" id="_dateTime_014">
+        <variable name="dateTime_014"/>
+        <literalExpression>
+            <text>date and time("2018-12-08T10:30:05") in (date and time("2018-12-08T10:30:04"), >=date and time("2018-12-08T10:30:05"))</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="dateTime_014_a" id="_dateTime_014_a">
+        <variable name="dateTime_014_a"/>
+        <literalExpression>
+            <text>date and time("2018-12-08T10:30:05") in (date and time("2018-12-08T10:30:04"), >=date and time("2018-12-08T10:30:06"))</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_001" id="_list_001">
+        <variable name="list_001"/>
+        <literalExpression>
+            <text>[1,2,3] in [[1,2,3,4], [1,2,3]]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_001_a" id="_list_001_a">
+        <variable name="list_001_a"/>
+        <literalExpression>
+            <text>[1,2,3,5] in [[1,2,3,4], [1,2,3]]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_011" id="_list_011">
+        <variable name="list_011"/>
+        <literalExpression>
+            <!-- looks like it should be an equals form left to right .. but the list [1,2,3] does not
+            equal 1, 2 or 3] values -->
+            <text>[1,2,3] in [1,2,3]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_011_a" id="_list_011_a">
+        <variable name="list_011_a"/>
+        <literalExpression>
+            <text>[1,2,3] in [[1,2,3], [1,2,3,4]]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_011_b" id="_list_011_b">
+        <variable name="list_011_b"/>
+        <literalExpression>
+            <text>[1,2,2] in [[1,2,3], [1,2,3,4]]</text>
+        </literalExpression>
+    </decision>
+
+<!--
+    <decision name="list_013" id="_list_013">
+        <variable name="list_013"/>
+        <literalExpression>
+            <text>[1,2,3] in ? = [1,2,3]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_013_a" id="_list_013_a">
+        <variable name="list_013_a"/>
+        <literalExpression>
+            <text>[1,2,3] in ? != [1,2,3]</text>
+        </literalExpression>
+    </decision>
+-->
+
+    <decision name="list_014" id="_list_014">
+        <variable name="list_014"/>
+        <literalExpression>
+            <text>[1,2,3] in ([1,2,3,4], [1,2,3])</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_014_a" id="_list_014_a">
+        <variable name="list_014_a"/>
+        <literalExpression>
+            <text>[1,2,3] in ([[1,2,3,4]], [[1,2,3,5]])</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="list_014_b" id="_list_014_b">
+        <variable name="list_014_b"/>
+        <literalExpression>
+            <text>[1,2,3] in ([[1,2,3,4]], [[1,2,3]])</text>
```

Comment:
Rule 51.d as tested for instance by number_014
We apply the element, a list `[1,2,3]`, to a list of positive unary tests { `=[[1,2,3,4]]` , `=[[1,2,3]]`} .
Results are `{false, false}`.
End result, `false`.

Reply:
@tarilabs. test updated as such.